src/pages/Storefront/Cart.js

```diff
@@ -137,15 +125,15 @@ const OrderSummary = ({ checkoutData }) => {
           </chakra.h4>
         </VStack>
         <Divider borderColor="brand.gray" />
-        <Flex w="100%" justifyContent="space-between">
+        <HStack w="100%" justifyContent="space-between" spacing={4}>
           <Heading as="h4" size="subtitle" textTransform="uppercase">
-            estimated total
+            Estimated Total
           </Heading>
           <Heading as="h4" size="subtitle">{`$${totalPrice}`}</Heading>
-        </Flex>
+        </HStack>
       </VStack>
-      <Link href={webUrl}>
-        <Button size="md" textTransform="uppercase">
+      <Link href={webUrl} w="100%">
+        <Button size="md" textTransform="uppercase" minW="100%">
           proceed to checkout
```

Comment:
Maybe change to `Proceed to Checkout` for consistency? doesn't really matter since there's `textTransform` anyways but I see you did it above

Reply:
fixedin dd4daf13eb